### ISSUE TITLE
Missing pac4j v6 Support

### ISSUE THREAD

**Issue by JapuDCret** [pac4j/spring-webflux-pac4j#49]
Out-of-the-box I struggle to get spring-webflux-pac4j working with Pac4J v6

For example, there is no provided implementation of `WebContextFactory` like there is for `JEEFrameworkParameters` to expose the WebFlux `ServerWebExchange`

**Comment by leleuj** [comment]
Indeed, I haven't worked yet on upgrading this implementation to use pac4j v6.

The `SpringWebfluxWebContextFactory` implements the `WebContextFactory ` interface.

**Comment by JapuDCret** [comment]
Thanks for the quick response, I'm trying to provide a PR for this here.

However, I'm struggling with the removal of the `FindBest` class, which has been heavily utilized.
Is there some already migrated project, which I could look into?

**Comment by leleuj** [comment]
Indeed, the `FindBest` class has been removed and the change is not obvious.

Before: https://github.com/pac4j/jee-pac4j/blob/7.1.x/javaee-pac4j/src/main/java/org/pac4j/jee/filter/CallbackFilter.java#L65

After: https://github.com/pac4j/jee-pac4j/blob/master/javaee-pac4j/src/main/java/org/pac4j/jee/filter/CallbackFilter.java#L61

Now, default settings must be set by the appropriate `FrameworkAdapter`.

This is the behavior of the `FrameworkAdapter`: https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/adapter/FrameworkAdapter.java#L23

For SpringWebflux, you should create a `org.pac4j.jee.adapter.JEEFrameworkAdapter`. Like: https://github.com/pac4j/pac4j/blob/master/pac4j-javaee/src/main/java/org/pac4j/jee/adapter/JEEFrameworkAdapter.java#L44, certainly without the `compareManagers` method and with the appropriate `applyDefaultSettingsIfUndefined` method.
